blueocean-dashboard/src/main/js/components/karaoke/components/InputStep.jsx

```diff
@@ -8,6 +8,7 @@ import {
     StringUtil,
     logging,
 } from '@jenkins-cd/blueocean-core-js';
+import {rootPath} from '../../../util/UrlUtils';
```

Comment:
`buildClassicInputUrl` didn't build the correct URL and I was not sure it was not being used somewhere else and that is why I decided to not fix that function so I wouldn't break a functionality somewhere else. I also saw that in other places the URL is being built on the fly so I just replicated that behaviour (https://github.com/jenkinsci/blueocean-plugin/blob/master/blueocean-dashboard/src/main/js/components/RunDetails.jsx#L46)

If you think that function is not being used anywhere I can fix and use that but my opinion is that given how few lines of code this requires, I would leave the code here for easier debugging in the future (and because it is only being used here, there is no re-usability), it is a little hard to track those functions down all the way in core-js.

Reply:
The problem with all this "other places the URL is being built" is that you have to scan the whole wrap in case something in the url pattern changes.  

https://github.com/jenkinsci/blueocean-plugin/search?utf8=%E2%9C%93&q=buildClassicInputUrl&type= it is ATM not used anymore, so I think the save bet is to fix the method and then use it.